mmocr/models/textdet/dense_heads/east_head.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.cnn import ConvModule
+from mmcv.runner import BaseModule, Sequential
+
+from mmocr.models.builder import HEADS
+from .head_mixin import HeadMixin
+
+
+@HEADS.register_module()
+class EASTHead(HeadMixin, BaseModule):
+
+    def __init__(self,
+                 box_type,
+                 num_classes,
+                 in_channels,
+                 loss=dict(type='EASTLoss', box_type='RBOX'),
+                 postprocessor=dict(
+                     type='EASTPostprocessor',
+                     text_repr_type='quad',
+                     box_type='RBOX'),
+                 init_cfg=[
+                     dict(type='Kaiming', layer='Conv'),
+                     dict(
+                         type='Constant', layer='BatchNorm', val=1., bias=1e-4)
+                 ],
+                 train_cfg=None,
+                 test_cfg=None):
+        BaseModule.__init__(self, init_cfg=init_cfg)
+        HeadMixin.__init__(self, loss, postprocessor)
+        assert box_type in ['RBOX', 'QUAD']
+        assert box_type == loss['box_type'] == postprocessor['box_type']
+        self.box_type = box_type
+        self.num_classes = num_classes
+        self.in_channels = in_channels
+        self.train_cfg = train_cfg
+        self.test_cfg = test_cfg
+        self._init_layers()
+
+    def _init_layers(self):
+        """Initialize layers of the head."""
+        self._init_cls_convs()
+        self._init_reg_convs()
```

Comment:
These `_init_layers`, `_init_cls_convs`, and `_init_reg_convs` look a little bit weird here. Is it necessary to split these definitions into two functions? When I look at the names, my first impression is that these functions are used for initializing weights. Directly defining `self.score_conv` and `self.rec_conv` in `__init__` might be enough

Reply:
You are right, This is indeed weird.